hoodie-client/src/test/java/com/uber/hoodie/common/TestRawTripPayload.java

```diff
@@ -91,6 +97,14 @@ public String getPartitionPath() {
         }
     }
 
+    @Override
+    public Map<String, String> getMetadata() {
+        Map<String, String> metadataMap = new HashMap<>();
```

Comment:
Can you come up with some reasonable values here for the map? Current values don't convey your usage pattern well IMO